omniledger/darc/darc_test.go

```diff
@@ -145,6 +145,71 @@ func TestDarc_EvolveMore(t *testing.T) {
 	require.NotNil(t, darcs[len(darcs)-2].Verify())
 }
 
+func TestDarc_EvolveMoreOnline(t *testing.T) {
+	d := createDarc(1, "testdarc").darc
+	require.Nil(t, d.Verify())
+	prevOwner := NewSignerEd25519(nil, nil)
+	require.Nil(t, d.Rules.UpdateEvolution(
+		expression.InitOrExpr(prevOwner.Identity().String())))
+
+	darcs := []*Darc{d}
+	for i := 0; i < 10; i++ {
+		dNew := darcs[len(darcs)-1].Copy()
+		dNew.IncrementVersion()
+		newOwner := NewSignerEd25519(nil, nil)
+		require.Nil(t, dNew.Rules.UpdateEvolution([]byte(newOwner.Identity().String())))
+		require.Nil(t, dNew.Evolve(darcs, prevOwner))
+		// require.Nil(t, dNew.Verify())
```

Comment:
Why is that commented out?

Reply:
probably as a result of debugging, it's uncommented now